compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+
+services:
+ client:
+  build:
+   context: ./client
+  ports:
+   - 3000:3000
```

Comment:
Burde den ikke udstille port 80? Så slipper brugeren for at skrive :3000 til url